### PR TITLE
Chore: Update @azure/msal-browser v2 to v3

### LINKED ISSUE #2824: Update from msal-browser v2 to v3
This is a major update that involves breaking changes well documented here : [Migrating from MSAL 2.x to MSAL 3.x](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/v2-migration.md)

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview
Fixes #2824

### CODE REVIEW

File: src/app/services/actions/permissions-action-creator.spec.ts [hunk 1]
```diff
@@ -432,8 +438,8 @@ describe('Permissions action creators', () => {
         {
           type: 'QUERY_GRAPH_STATUS',
           response: {
-            statusText: 'Failed',
-            status: 'An error occurred when unconsenting. Please try again',
+            statusText: 'Unable to dissent',
+            status: 'Unable to dissentYou require the following permissions to revoke',
```
Comment (Onokaev) [inline src/app/services/actions/permissions-action-creator.spec.ts:442]: Nit: Space between `dissent.` and `You`

Comment (ElinorW) [reply src/app/services/actions/permissions-action-creator.spec.ts:442]: The test fails if I add the space because that's how the error is being passed

**Review by thewahome — DISMISSED** [review]

File: src/app/services/actions/permissions-action-creator.spec.ts [hunk 2]
```diff
@@ -432,8 +438,8 @@ describe('Permissions action creators', () => {
         {
           type: 'QUERY_GRAPH_STATUS',
           response: {
-            statusText: 'Failed',
-            status: 'An error occurred when unconsenting. Please try again',
+            statusText: 'Unable to dissent',
```
Comment (thewahome) [inline src/app/services/actions/permissions-action-creator.spec.ts:441]: Are these strings localised anywhere down the line?

**Review by thewahome — DISMISSED** [review]

File: src/app/services/actions/permissions-action-creator.spec.ts (hunk 2 above)
Comment (ElinorW) [reply src/app/services/actions/permissions-action-creator.spec.ts]: No, they were not...
I've added them to the localisation file

**Review by thewahome — APPROVED** [review]

**Pull request merged**